.github/workflows/build-test.yml

```diff
@@ -69,68 +59,10 @@ jobs:
         with:
           access_token: ${{ github.token }}
 
-  pick_environment:
-    name: Pick Environment
-    runs-on: ubuntu-latest
-    outputs:
-      vsix_name: ${{ env.vsix_name }}
-      test_matrix_os: ${{ env.test_matrix_os }}
-      release_channel: ${{ env.release_channel }}
-      package_command: ${{ env.package_command }}
-    if: github.repository == 'microsoft/vscode-jupyter'
-    steps:
-      - name: dump_event
-        uses: crazy-max/ghaction-dump-context@v1
-
-      - name: default_package
-        run: |
-          echo "package_command=package" >> $GITHUB_ENV
-
-      - name: pr
-        if: github.event_name == 'pull_request'
-        run: |
-          echo "vsix_name=ms-toolsai-jupyter-insiders.vsix" >> $GITHUB_ENV
-          echo "test_matrix_os=[\"ubuntu-latest\", \"windows-latest\"]" >> $GITHUB_ENV
-
-      - name: pr release push
```

Comment:
We don't need any of these anymore as we now use the azure pipelines for releases.